content/courses/cmpe470.en.md

```diff
@@ -7,9 +7,40 @@ aliases:
   - undergraduate/courses/cmpe470
 ---
 
+## Course Information
+
+<!-- prettier-ignore-start -->
+{{< table class="table-hover table-sm" >}}
+|||
+| :-- | :-- |
+| Faculty | Faculty of Engineering |
+| Course Code | CMPE470 |
+| Course Title | Computer Performance Evaluation |
+| Language of Instruction | English |
+| Course Semester | -- |
```

Comment:
Not written in the [reference](https://www.bogazici.edu.tr/tr_TR/Content/Akademik/Lisans_Katalogu/Muhendislik_Fakultesi/Bilgisayar_Muhendisligi_Bolumu)

Reply:
I actually did checked the [reference page](https://www.bogazici.edu.tr/tr_TR/Content/Akademik/Lisans_Katalogu/Muhendislik_Fakultesi/Bilgisayar_Muhendisligi_Bolumu) you shared and put that info already. However that info is not helping me to fill spaces like course semester or outcome of the course

Reply:
According to the latest information I got from my friends, the 470 course has not been open for years. I'm not sure the page is supposed to exist.